help/access/file-shares.md

```diff
@@ -1,302 +1,237 @@
----
-title: File shares. File Manager
----
+# Files
 
-**File Manager** lets you work with files and directories on your system from the convenience of your web browser. You
-can browse, preview, open, create, delete, rename, download, clone, and share files and directories.
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
 
-To open **File Manager**, from the main menu, select **Data**>**Files**.
+Datagrok offers a convenient interface for working with files. You can connect to [popular file systems](supported-connectors.md), including the [Amazon S3 bucket](connectors/s3.md), [Dropbox](connectors/dropbox.md), [Google Drive](connectors/googlecloud.md), and [Git](connectors/git.md), as well as [Windows and Linux network shares](connectors/files.md). In addition, each Datagrok user has a personal directory called **Home** for uploading files using the drag-and-drop feature. Datagrok automatically creates this directory upon signup.
```

Comment:
for uploading files using the drag-and-drop feature
=>
for keeping their files